src/saveload/saveload.h

```diff
@@ -359,17 +360,27 @@ enum SavegameType {
 	SGT_INVALID = 0xFF, ///< broken savegame (used internally)
 };
 
+enum SavegameFlags : uint8 {
+	SGF_NO_THREAD  = 0,       ///< Do not use threads.
+	SGF_THREADED   = 1 << 0,  ///< Save threaded.
+	SGF_PERSISTENT = 0,       ///< Savegame should persistent through time and different clients.
+	SGF_TEMPORARY  = 1 << 1,  ///< Savegame will only be used temporary or locally.
```

Comment:
the fact that the enum contains duplicate values with entirely different meanings is a bit odd

Reply:
They are toggles. The idea is that you either use`NO_THREAD` or `THREADED`. We could do away with the 0-value ones, but than it is harder to see how a savegame is made and easier to think either one is on-by-default.

If you have other suggestions, I am all ears :)

Reply:
Alternatively, I can do `0 << 0` and `0 << 1` to make it more clear they are toggles? Not sure that helps :D